### ISSUE TITLE
can't install in rapsberry 3 strectch. Unable to locate package deconz

### ISSUE THREAD

**Issue by Klagio** [dresden-elektronik/deconz-rest-plugin#1639]
```

pi@pihole:~ $ wget -O - http://phoscon.de/apt/deconz.pub.key | \
>            sudo apt-key add -
--2019-06-29 08:03:12--  http://phoscon.de/apt/deconz.pub.key
Resolving phoscon.de (phoscon.de)... 81.169.237.235
Connecting to phoscon.de (phoscon.de)|81.169.237.235|:80... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1692 (1.7K) [application/octet-stream]
Saving to: ‘STDOUT’

-                             100%[=================================================>]   1.65K  --.-KB/s    in 0s

2019-06-29 08:03:12 (49.6 MB/s) - written to stdout [1692/1692]

OK

pi@pihole:~ $ sudo apt update
Hit:1 http://archive.raspberrypi.org/debian stretch InRelease
Hit:2 http://phoscon.de/apt/deconz stretch InRelease
Get:3 http://deb.debian.org/debian unstable InRelease [214 kB]
Hit:4 https://repo.mosquitto.org/debian stretch InRelease
Get:5 http://download.zerotier.com/debian/stretch stretch InRelease [20.0 kB]
Err:3 http://deb.debian.org/debian unstable InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7638D0442B90D010 NO_PUBKEY 04EE7237B7D453EC
Hit:6 http://raspbian.raspberrypi.org/raspbian stretch InRelease
Reading package lists... Done
W: GPG error: http://deb.debian.org/debian unstable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7638D0442B90D010 NO_PUBKEY 04EE7237B7D453EC
E: The repository 'http://deb.debian.org/debian unstable InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
pi@pihole:~ $ sudo apt install deconz
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package deconz
pi@pihole:~ $

```

**Comment by Klagio** [comment]
Nothing, I can't install deCONZ on a raspberrry pi3 following the documentation

**Comment by manup** [comment]
>  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 7638D0442B90D010 NO_PUBKEY 04EE7237B7D453EC 

Doesn't look deCONZ specific. It seems that your system is is configured for unstable Debian repositories and is missing pubic keys to verify these packages.

You need to add these as described here (replace with the keys from the error log):

https://ccm.net/faq/809-debian-apt-get-no-pubkey-gpg-error
